ci-operator/config/maistra/envoy/maistra-envoy-maistra-2.2.yaml

```diff
@@ -0,0 +1,26 @@
+build_root:
+  image_stream_tag:
+    name: maistra-builder
+    namespace: ci
+    tag: "2.2"
+  use_build_cache: true
+resources:
+  '*':
+    limits:
+      cpu: "8"
+      memory: 16Gi
+    requests:
+      cpu: "4"
+      memory: 4Gi
+tests:
+- as: unit
+  commands: |
+    export CI="true"
+    export BAZEL_DISK_CACHE="/tmp/cache"
```

Comment:
Is this cache an actual cache, shared with other runners and reused across builds?

If not, I suggest removing it, as adding a cold cache is worse than no cache at all.